### DISCUSSION TITLE
Rules in strict mode with a not-filter

### DISCUSSION THREAD

**Discussion by arjenfvellinga** [firefly-iii/firefly-iii#7317] [Firefly III]
![image](https://user-images.githubusercontent.com/16539832/229365018-0e562b83-ae46-4bdd-8a7b-1d1de01ff2eb.png)

In strict mode both triggers should be valid, and with the two I setup I would expect transactions that contain the word 'Vakantie' but not 'Spaargeld'. So I would not expect the matched transaction from the screenshot.

Is this this working correct?

**Comment by JC5** [comment]
I have no clue 😅 I'll do some digging.

**Comment by JC5** [comment]
The rule will work as expected, it's just that the preview is wrong.

**Comment by JC5** [comment]
And this will be fixed in the next patch release :+1:

**Reply by arjenfvellinga** [reply]
Great! thanks
